corehq/apps/accounting/forms.py

```diff
@@ -1993,77 +1996,39 @@ def send_message(self):
                                     email_from=settings.DEFAULT_FROM_EMAIL)
 
 
-class AnnualPlanContactForm(forms.Form):
-    name = forms.CharField(
-        label=gettext_noop("Name")
-    )
-    company_name = forms.CharField(
-        required=False,
-        label=gettext_noop("Company / Organization")
-    )
-    message = forms.CharField(
-        required=False,
-        label=gettext_noop("Message"),
-        widget=forms.Textarea(attrs={"class": "vertical-resize"})
-    )
+class EnterprisePlanContactForm(PlanContactForm):
+
+    def send_message(self):
+        subject_tag = "[Enterprise Plan Request]"
+        super().send_message(subject_tag)
+
+
+class AnnualPlanContactForm(PlanContactForm):
 
     def __init__(self, domain, web_user, on_annual_plan, data=None, *args, **kwargs):
-        self.domain = domain
-        self.web_user = web_user
-        super(AnnualPlanContactForm, self).__init__(data, *args, **kwargs)
-        from corehq.apps.domain.views.accounting import (
-            DomainSubscriptionView,
-            SelectPlanView,
-        )
-        self.helper = FormHelper()
-        self.helper.label_class = 'col-sm-3 col-md-2'
-        self.helper.field_class = 'col-sm-9 col-md-8 col-lg-6'
-        self.helper.form_class = "form-horizontal"
+        from corehq.apps.domain.views.accounting import DomainSubscriptionView
```

Comment:
Seeing the local import was a smell for me. This would be a reason to allow the caller to pass in the entire URL, because then this Form doesn't need to be aware of anything from the View.